integrations/adobe-analytics/lib/index.js

```diff
@@ -1456,6 +1462,19 @@ function createStandardVideoMetadata(track, mediaObj) {
   );
 }
 
+function createCustomVideoMetadataContext(track, options) {
```

Comment:
Should we be applying the `Context Data Property Prefix` setting here?

Reply:
Hey @gpsamson this one is a bit confusing. So the reason I left it out of the Heartbeat is because for non-heartbeat events we rely on the Context Data Variables setting mappings to identify which properties in the context object a customer also wants to send as a "Context Data Variable". Inherently we will send all the properties in the property object so they don't need to add mappings for properties. The "Prefix" is used in case there is some kind of nesting within the Context object of the payload.  

There is confusion in terms of how the "Context Data Variables" setting interacts with the Adobe Heartbeat implementation. Per this specific request, Fox is expecting that if they indicate mappings in the "Context Data Variables" that for video events using the heartbeat we will pull that data from the **properties object** (not context) and send it to AA as custom video metadata. We currently don't document at all how customers are supposed to sent custom metadata just that they can, so this is definitely one of those areas for improvement. I left the "prefix" out because we are not looking in the context object though since video metadata is sent in properties.

Reply:
Let me know if I can clarify anything there.

Reply:
Got it, makes sense. Thanks for clarifying!